internal/pkg/action/operator_uninstall.go

```diff
@@ -74,10 +65,20 @@ func (u *OperatorUninstall) Run(ctx context.Context) error {
 		return fmt.Errorf("get subscription CSV %q: %v", csvName, err)
 	}
 
+	if csv == nil {
+		return fmt.Errorf("get subscription CSV %q: %v", csvName, err)
+	}
```

Comment:
I'm not sure I agree with this being an error condition. See https://github.com/operator-framework/kubectl-operator/blob/9e5901c77b37c9a709cedc080733d3ca7bb50b64/internal/pkg/action/operator_uninstall.go#L148-L153

Reply:
Fixed to return a log and a nil error instead